web/components/layout/tabs.tsx

```diff
@@ -1,82 +1,121 @@
 import clsx from 'clsx'
-import Link from 'next/link'
+import { useRouter, NextRouter } from 'next/router'
 import { ReactNode, useState } from 'react'
-import { Row } from './row'
 import { track } from '@amplitude/analytics-browser'
 
 type Tab = {
   title: string
   tabIcon?: ReactNode
   content: ReactNode
-  // If set, change the url to this href when the tab is selected
-  href?: string
   // If set, show a badge with this content
   badge?: string
 }
 
-export function Tabs(props: {
+type TabProps = {
   tabs: Tab[]
-  defaultIndex?: number
   labelClassName?: string
   onClick?: (tabTitle: string, index: number) => void
   className?: string
   currentPageForAnalytics?: string
-}) {
+}
+
+export function UncontrolledTabs(props: TabProps & { activeIndex: number }) {
```

Comment:
I think you have the terminology backward re: Controlled vs Uncontrolled components

Controlled components are "controlled" by a parent, and don't store any state.
https://stackoverflow.com/a/42522792/1592933

Reply:
How embarrassing! In my mind it was like, controlled components control themselves, and uncontrolled components need something else to control them. I guess it's probably not the best jargon ever invented.